pydis_site/apps/staff/tests/test_logs_view.py

```diff
@@ -0,0 +1,160 @@
+from django.test import Client, TestCase
+from django.utils import timezone
+from django_hosts.resolvers import reverse, reverse_host
+
+from pydis_site.apps.api.models.bot import DeletedMessage, MessageDeletionContext, Role, User
+from ..templatetags.deletedmessage_filters import hex_colour
```

Comment:
Feels like this could be an absolute import?

Reply:
Sure. I'll change it. I've used a relative one because it's a within-app import.